src/components/featureHighlight/index.tsx

```diff
@@ -35,96 +151,15 @@ const contentViewHeight = Constants.isAndroid ? 268 : 282;
  * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/FeatureHighlightScreen.js
  */
 /*eslint-enable*/
-class FeatureHighlight extends BaseComponent {
+class FeatureHighlight extends Component<FeatureHighlightProps, State> {
   static displayName = 'FeatureHighlight';
-  static propTypes = {
-    /**
-     * Boolean to determine if to present the feature highlight component
-     */
-    visible: PropTypes.bool.isRequired,
-    /**
-     * Frame of the area to highlight {x, y, width, height}
-     */
-    highlightFrame: PropTypes.shape({
-      x: PropTypes.number,
-      y: PropTypes.number,
-      width: PropTypes.number,
-      height: PropTypes.number
-    }),
-    /**
-     * Callback that extract the ref of the element to be highlighted
-     */
-    getTarget: PropTypes.func,
-    /**
-     * Title of the content to be displayed
-     */
-    title: PropTypes.string,
-    /**
-     * Message to be displayed
-     */
-    message: PropTypes.string,
-    /**
-     * Title text style
-     */
-    titleStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
-    /**
-     * Message text style
-     */
-    messageStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
-    /**
-     * Title's max number of lines
-     */
-    titleNumberOfLines: PropTypes.number,
-    /**
-     * Message's max number of lines
-     */
-    messageNumberOfLines: PropTypes.number,
-    /**
-     * Props that will be passed to the dismiss button
-     */
-    confirmButtonProps: PropTypes.object,
-    /**
-     * Callback for the background press
-     */
-    onBackgroundPress: PropTypes.func,
-    /**
-     * Color of the content's background (usually includes alpha for transparency)
-     */
-    overlayColor: PropTypes.string,
-    /**
-     * Color of the content's text
-     */
-    textColor: PropTypes.string,
-    /**
-     * Color of the border around the highlighted element
-     */
-    borderColor: PropTypes.string,
-    /**
-     * Width of the border around the highlighted element
-     */
-    borderWidth: PropTypes.number,
-    /**
-     * Border radius for the border corners around the highlighted element
-     */
-    borderRadius: PropTypes.number,
-    /**
-     * The minimum size of the highlighted component (Android API 21+, and only when passing a ref in 'getTarget')
-     */
-    minimumRectSize: PropTypes.shape({
-      width: PropTypes.number,
-      height: PropTypes.number
-    }),
-    /**
-     * The padding of the highlight frame around the highlighted element's frame (only when passing ref in 'getTarget')
-     */
-    innerPadding: PropTypes.number,
-    /**
-     * PageControl component's props
-     */
-    pageControlProps: PropTypes.shape(PageControl.propTypes)
-  };
+  static propTypes = {};
```

Comment:
Why did you leave the `propTypes` static object?

Reply:
removed